dev/traffic_router/run.sh

```diff
@@ -18,7 +18,7 @@
 
 set -o errexit
 
-cd "$TC/traffic_router"
+cd "$TC/dev/traffic_router"
 
 mvn -Dmaven.test.skip=true compile package -P \!rpm-build
```

Comment:
The working directory should still be `$TC/traffic_router` for building TR

Reply:
:facepalm: